### PR TITLE
container storybook docs and fix

### PR DESCRIPTION
**Pull request by chrisolsen**

### CODE REVIEW

**Comment by bluekurinji** [comment]
Every component display need to have code sample 

image

image

**Comment by bluekurinji** [comment]
`<h2>Custom H2</h2>` Is not visible component 

image

File: libs/docs/src/components/common/Container.stories.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { Meta, Story } from '@storybook/addon-docs';
-import { Tabs, Tab, Note, CodeSnippet } from '@abgov/shared/storybook-common';
+import { Tabs, Tab, Note, CodeSnippet, SupportInfo } from '@abgov/shared/storybook-common';
 import { GoAContainer, GoAButton, GoABadge } from '@abgov/react-components';
```
[the remark this answers is not in the document]
Comment (chrisolsen) [reply libs/docs/src/components/common/Container.stories.mdx:4]: ?

**Comment by bluekurinji** [comment]
> Every component display need to have code sample
> 
> image
> 
> image

This one

**Review by bluekurinji — APPROVED** [review]

**Pull request merged**